pkgs/development/python-modules/contexttimer/default.nix

```diff
@@ -0,0 +1,20 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "contexttimer";
+  version = "0.3.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "35a1efd389af3f1ca509f33ff23e17d98b66c8fde5ba2a4eb8a8b7fa456598a5";
+  };
+
```

Comment:
Missing tests and pythonImportsCheck.

Reply:
I added the pythonImportsCheck, but the pypi packages doesnt come with the test file, I don't think we should bother.

Reply:
Then please switch to the source release. We should bother because pythonPackages without any tests easily bitrot and break with dependency updates.

Reply:
hm, ok, I will try.

Reply:
Done, I had to patch the test but it worked!